script/tool/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## 13.1
```

Comment:
Oops. (Since this isn't in packages/ it doesn't get the metadata checks.)

Reply:
Does script/tool still need to be versioned like this?

Reply:
I publish it periodically so that flutter/packages can use the latest updates, so it does need versioning. Once the repo merge is done it won't.